### PR TITLE
Allow updating and deleting guides, fixes #113, fixes #109, fixes #115

### LINKED ISSUE #109: Add guide delete option

### LINKED ISSUE #113: Add ability to edit the guide
I created a fold, worked on it, saved it - it has been created, great, but then I open it, click save, and I'd expect it to be updated, but that creates a new one instead.

Related: #115

### LINKED ISSUE #115: Multiple save options in the create view.
When not logged in, only "save to file". When logged in, "save to file", and "save" (which saves it on my account).

Actually I think, we should have also the option: "save", "save as". So as to be able to edit the guide or save it as a new one if we wish.

So save, save as.. and then file/account option underneath.

Related: #113

### PR DESCRIPTION
**Pull request by maciekmm**

### CODE REVIEW

File: frontend/src/app/guide-creator/index.js [hunk 1]
```diff
@@ -14,9 +14,37 @@ import {
...
+import { modelToBase64 } from "../../download"
+import { useSnackbar } from "notistack"
+import { useCommunityStore } from "@store/community"
 
 export default function GuideCreator() {
 	const [{ model, frame, selectedEdges }, dispatch] = useCreatorStore()
+	const [{ userId }] = useCommunityStore()
+
+	const { guideId } = useParams()
+	const isEditingGuide = guideId !== undefined && guideId !== null
```
Comment (zemiret) [inline frontend/src/app/guide-creator/index.js:28]: guideId != null does the same

File: frontend/src/store/creator/reducer.js [hunk 2]
```diff
@@ -2,28 +2,31 @@ import {
...
+import preprocessFOLDModel, {
+	markFramesToInheritDeeply,
+} from "@fold/preprocess"
+import penguinGuide from "../../../../assets/origuide-models/penguin.fold"
 
 export const initialState = {
 	...viewerInitialState,
 	model: (() => {
-		const model = viewerInitialState.model
+		const model = preprocessFOLDModel(penguinGuide)
```
Comment (zemiret) [inline frontend/src/store/creator/reducer.js:29]: Is this here on purpose?

**Review by zemiret — CHANGES_REQUESTED** [review]
It's ok. Just this 1 penguin guide. Is this on purpose? If yes, you can merge.

File: frontend/src/store/creator/reducer.js (hunk 2 above)
Comment (maciekmm) [reply frontend/src/store/creator/reducer.js:29]: Yes, it makes little sense to load a folded model into the "Create" window.

File: frontend/src/app/guide-creator/index.js (hunk 1 above)
Comment (maciekmm) [reply frontend/src/app/guide-creator/index.js:28]: Sure,  but the linter will complain about not using triple equals.

**Pull request merged**